helm-charts/falcon-sensor/values.schema.json

```diff
@@ -14,6 +14,10 @@
                     "example": [
```

Comment:
Does `registryConfigJSON` need to be added to the schema?

Reply:
Looking into how to best do that because it is multi-object right now (null and string). The schema definitely needs to be updated for more things, but that is a future finalization PR.

Reply:
Wasn't sure if helm supported the json schema fully. Looks like it does. Yay! Fixed.